src/injected/all-frame-runner.ts

```diff
@@ -66,12 +66,12 @@ export class AllFrameRunner<T> {
     }
 
     public start = async () => {
-        this.listener.start();
+        await this.listener.start();
```

Comment:
Perhaps less risky to keep the execution order as-is? ie, invoke listener.start, then await sendCommand, and only then await the promise returned by listener.start?

Reply:
(same for stop)